### ISSUE TITLE
will label shift eventually cause performance drop in test set?

### ISSUE THREAD

**Issue by yaoqi-zd** [blindauth/labelshiftexperiments#5]
suppose we train a binary classifier on a dataset that has 80% positive and 20% negative, the trained classifier(may not be well trained) tends to classifier all samples as positive. If the test set all are positive, we can get 100% test accuracy.  In this case, do we need to adjust the prediction?

more generally, if we train a classifier on a imbalanced dataset with cross entropy loss, it seems that the classifier tends to classify sample as head classes, so when the test set has a larger proportion of head classes, the test accuracy may even grow, if the test set has more tail class samples, the test accuracy will drop a lot. So do we always need to adjust the prediction?
